tests/functional/tests/osd/test_osds.py

```diff
@@ -4,30 +4,45 @@
 class TestOSDs(object):
 
     @pytest.mark.no_docker
-    def test_ceph_osd_package_is_installed(self, node, Package):
-        assert Package("ceph-osd").is_installed
+    def test_ceph_osd_package_is_installed(self, node, host):
+        assert host.package("ceph-osd").is_installed
 
-    def test_osds_listen_on_public_network(self, node, Command):
+    def test_osds_listen_on_public_network(self, node, host):
         # TODO: figure out way to paramaterize this test
         nb_port = (node["num_devices"] * 2)
-        assert Command.check_output("netstat -lntp | grep ceph-osd | grep %s | wc -l" % (node["address"])) == str(nb_port)
+        assert host.check_output("netstat -lntp | grep ceph-osd | grep %s | wc -l" % (node["address"])) == str(nb_port)
 
-    def test_osds_listen_on_cluster_network(self, node, Command):
+    def test_osds_listen_on_cluster_network(self, node, host):
         # TODO: figure out way to paramaterize this test
         nb_port = (node["num_devices"] * 2)
-        assert Command.check_output("netstat -lntp | grep ceph-osd | grep %s | wc -l" % (node["cluster_address"])) == str(nb_port)
+        assert host.check_output("netstat -lntp | grep ceph-osd | grep %s | wc -l" % (node["cluster_address"])) == str(nb_port)
 
-    def test_osd_services_are_running(self, node, Service):
+    def test_osd_services_are_running(self, node, host):
         # TODO: figure out way to paramaterize node['osds'] for this test
         for osd in node["osds"]:
-            assert Service("ceph-osd@%s" % osd).is_running
+            assert host.service("ceph-osd@%s" % osd).is_running
+
+    def test_osd_services_are_running(self, host):
+        # TODO: figure out way to paramaterize node['osds'] for this test
+        for osd in node["osds"]:
+            assert host.service("ceph-osd@%s" % osd).is_running
 
     @pytest.mark.no_docker
-    def test_osd_are_mounted(self, node, MountPoint):
+    def test_osd_are_mounted(self, node, host):
         # TODO: figure out way to paramaterize node['osd_ids'] for this test
         for osd_id in node["osd_ids"]:
             osd_path = "/var/lib/ceph/osd/{cluster}-{osd_id}".format(
                 cluster=node["cluster_name"],
                 osd_id=osd_id,
             )
-            assert MountPoint(osd_path).exists
+            assert host.mount_point(osd_path).exists
+
+    def test_ceph_volume_is_installed(self, node, host):
+        if node['vars'].get('osd_scenario') != 'lvm':
```

Comment:
I think we should check for ``node['vars'].get('osd_scenario') != 'lvm'`` in ``conftest.py`` and do the skipping there. You can mark tests that only work for the ``lvm`` scenario with a decorator, this is how we do it for tests that only work with docker scenarios. 

https://github.com/ceph/ceph-ansible/blob/master/tests/functional/tests/mon/test_mons.py#L41

https://github.com/ceph/ceph-ansible/blob/master/tests/conftest.py#L28